portfolio/src/main/java/com/google/sps/Data/User.java

```diff
@@ -20,10 +20,12 @@
   private final boolean loggedIn;
   private final String authenticationUrl;
   private final String email;
+  private final String nickname;
```

Comment:
Can you add a comment here explaining what a nickname is? It's not immediately clear - is it a username for authentication? Is it a user-provided display name?

Reply:
It is a user-provided display name. Added the comment!